modules/reference/pages/systems/system-details/sd-states.adoc

```diff
@@ -1,63 +1,56 @@
 [[sd-states]]
-= SD States
+= States
```

Comment:
Too many small paras...

Reply:
lol, this seems to be a thing I do a lot :wink:

Reply:
With our previous proofreader, this would have caused you serious trouble :smiling_face_with_three_hearts:   And she was not wrong, I'd say :wink: